_playwright-tests/UI/SnapshotErrataCountAndFilter.spec.ts

```diff
@@ -115,7 +115,7 @@ test.describe('Snapshot Errata Count and Filter', () => {
       await snapshotListModal.getByRole('button', { name: 'Clear filters' }).click();
       await expect(
         snapshotListModal.getByRole('button', { name: 'Clear filters' }),
-      ).not.toBeVisible();
+      ).not.toBeVisible({ timeout: 1000 });
```

Comment:
I don't think this needs any adjustment. On line 115 you interact with "Clear filters" button, so it is visible and rendered. On line 118 you expect it not to be present. So that is all fine, you are waiting for it disappearing.
(This would be needed to be handled if the whole page would be reloaded, which would be actually a wrong behavior, but I think we are fine)

Reply:
I have reverted that change